app/scripts/templates/complete_reset_password.mustache

```diff
@@ -32,13 +32,17 @@
 
   {{#isLinkValid}}
   <header>
-    <h1 id="fxa-complete-reset-password-header">{{#t}}Reset password{{/t}}</h1>
+    <h1 id="fxa-complete-reset-password-header">{{#t}}Enter new password{{/t}}</h1>
   </header>
 
   <section>
     <div class="error"></div>
 
     <form novalidate>
+      <p class="reset-warning reset-warning-complete-password">
+          {{#t}}Once you submit your new password, you will lose any Sync data that is not on one of your devices.{{/t}}<a href="https://support.mozilla.org/en-US/products/firefox/sync" target="_blank">{{#t}}Learn how Sync works.{{/t}}</a>
```

Comment:
Remove the `en-US` from the link, SUMO will redirect to the appropriate locale based on the user's `Accept-Language` headers.

Can you break this line into two lines? I need to scroll to review it on GitHub.